### DISCUSSION TITLE
https://saicharankandukuri.github.io/posts/the-ppt-remorse/

### DISCUSSION THREAD

**Discussion by giscus** [SaicharanKandukuri/SaicharanKandukuri.github.io#4] [Blog]
# https://saicharankandukuri.github.io/posts/the-ppt-remorse/

A little story of how I was wrong about something I thought I was right about.

https://saicharankandukuri.github.io/posts/the-ppt-remorse/

**Comment by ayon-ssp** [comment]
![sad cat](https://i-cdn.embed.ly/1/display?key=fd92ebbc52fc43fb98f69e50e7893c13&url=http%3A%2F%2Fi.imgur.com%2FrolF402.gif%3Fnoredirect)
